test/integration/scheduler_perf/scheduler_perf_types.go

```diff
@@ -0,0 +1,28 @@
+package benchmark
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+		"k8s.io/kubernetes/pkg/api/v1"
+)
+
+// High Level Configuration for all predicates and priorities.
+type schedulerPerfConfig struct {
+	NodeAffinity *nodeAffinity
+	InterPodAffinity *interPodAffinity
+}
+
+// interPodAffinity priority configuration details.
+type interPodAffinity struct {
```

Comment:
@gmarek : I made the following assumptions regarding the comments you made:

- Top level structures like selector options (coming from metav1, v1 packages) need not be part of struct that we are creating so that we don't have to rely on them for changes made.
- PodAffinity struct has been deleted.(Initially we added it so that we can expand later.)